tests/test_telemetry.py

```diff
@@ -244,6 +278,9 @@ def validator(data):
 
         self.validate_library_telemetry_data(validator)
 
+    def setup_app_heartbeat(self):
+        time.sleep(20)
```

Comment:
Not the good way. You need at least 20 seconds of lifespan for the app, right? (IIRC, the heartbeat delay is set to 2 seconds)

Reply:
@mehulsonowal addressed this in his PR